prober/metrics.go

```diff
@@ -282,6 +284,61 @@ func collectFileMetrics(logger log.Logger, files []string, registry *prometheus.
 	return nil
 }
 
+func collectJKSMetrics(logger log.Logger, files []string, registry *prometheus.Registry, password string) error {
+	var (
+		totalCerts  []*x509.Certificate
+		jksNotAfter = prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: prometheus.BuildFQName(namespace, "", "jks_cert_not_after"),
+				Help: "NotAfter expressed as a Unix Epoch Time for a certificate found in a java keystore file",
+			},
+			[]string{"hostname", "file", "serial_no", "issuer_cn", "cn", "dnsnames", "ips", "emails", "ou"},
```

Comment:
Why does this need a `hostname` label? I'm not sure how that's related to a Java Keystore file.